docs/molnet_pr_template.md

```diff
@@ -0,0 +1,12 @@
+### Template for pull request contributing a new dataset to MoleculeNet
+Category: {Quantum Mechanics, Physical Chemistry, Biophysics, Physiology}  
```

Comment:
Should we add materials science in here since we have some new datasets/featurizers coming in?